### ISSUE TITLE
Inference problem

### ISSUE THREAD

**Issue by wulungching** [Stick-To/RefineDet-tensorflow#10]
Hello @Stick-To ,

Do you remember your training loss at the end of training process?
I trained model about 26 epoch on VOC2012, training loss was converged successfully. [pic](https://ppt.cc/fBbqPx)
But when I used one dog in picture to test model, it returned something weird. [sample](https://ppt.cc/fAMYHx)

**Comment by Stick-To** [comment]
![捕获](https://user-images.githubusercontent.com/34546552/63482552-de63d300-c4cb-11e9-9fd6-03257da31e80.PNG)

26 epoches is too small, change the hyper_parameters, like learning rate, when test change the nms threshold  etc.

**Comment by wulungching** [comment]
> ![捕获](https://user-images.githubusercontent.com/34546552/63482552-de63d300-c4cb-11e9-9fd6-03257da31e80.PNG)
> 
> 26 epoches is too small, change the hyper_parameters, like learning rate, when test change the nms threshold etc.

OK~ I will try this. Thanks a lot! :)
